Makefile

```diff
@@ -0,0 +1,18 @@
+GO ?= go
+GO_BUILD := $(GO) build
+
+.DEFAULT: bypass4netns
+
+bypass4netns:
+	$(GO_BUILD) -o $@ cmd/$@/main.go
+
+install: bypass4netns
+	install bypass4netns /usr/local/bin/bypass4netns
+
+uninstall:
+	rm -rf /usr/local/bin/bypass4netns
+
+clean:
+	rm -rf bypass4netns
+
+.PHONY: clean install
```

Comment:
`bypass4netns` should be marked as `.PHONY` too, because we do not detect the dependency of `bypass4netns` currently.